package.json

```diff
@@ -31,33 +31,29 @@
     "build": "rollup -c",
     "commit": "cz",
     "dev": "node dev.js",
-    "lint": "eslint . --fix --ignore-path .gitignore",
-    "lint:check": "eslint . --ignore-path .gitignore",
+    "lint": "eslint . --fix",
+    "lint:check": "eslint .",
     "test": "vitest"
   },
   "keywords": [],
   "author": "@djh",
   "license": "Apache-2.0",
   "dependencies": {
-    "@unocss/core": "0.x",
-    "@unocss/rule-utils": "0.x",
-    "@unocss/preset-mini": "0.x"
+    "@unocss/core": "~0.65.1",
+    "@unocss/preset-mini": "~0.65.1",
+    "@unocss/rule-utils": "~0.65.1"
   },
   "devDependencies": {
-    "@rollup/plugin-node-resolve": "^15.2.3",
-    "@semantic-release/changelog": "^6.0.2",
-    "@semantic-release/git": "^10.0.1",
-    "@unocss/autocomplete": "^0.58.5",
-    "@warp-ds/eslint-config": "1.0.5",
-    "cz-conventional-changelog": "^3.3.0",
-    "eslint": "8.57.0",
-    "prettier": "3.2.5",
-    "rollup": "^4.10.0",
-    "semantic-release": "^23.0.0",
-    "vitest": "^1.2.2"
-  },
-  "eslintConfig": {
```

Comment:
Why did you need to drop this extension and add a `eslint.config.js` instead?

Reply:
Moved from eslint 8.x to 9.x :)